### PR TITLE
adding proposed blog post on data licensing

### PR DESCRIPTION
**Pull request by elliewix**
Per email conversation, adding a PR for new blog post on data licensing.

Unsure about the meta area...

### CODE REVIEW

**Comment by gvwilson** [comment]
Looks good to me - thanks!

**Comment by tracykteal** [comment]
Looks great, thank you! 

So people know this is coming from someone with data curation expertise and perspective, would you be OK with adding your position in the 'authors' section, so: 

"Elizabeth Wickes, Data Curation Specialist, University of Illinois at Urbana-Champaign"

**Comment by elliewix** [comment]
Sure, but if we're going to add that stuff in, I'd like to add in a reference to the fact that I work in the library

So maybe

Elizabeth Wickes, Data Curation Specialist (University Library), University of Illinois at Urbana-Champaign

**Comment by tracykteal** [comment]
Yes, great, definitely however you want to have your affiliation.

**Comment by elliewix** [comment]
Okay, added in...I think. <githubhedge/>

**Comment by tracykteal** [comment]
All good!

**Comment by hlapp** [comment]
Per se there is nothing wrong with what the text says. However, I have a number of issues with the blog post as a whole, which aren't about the things that it says but about the things that it doesn't say but that people are frequently confused or misinformed about.

1. One of the most common misconceptions when to comes licensing data is that the vast majority of data people deal in science deal with are facts of nature and as such are not eligible for copyright protection. Hence, subjecting it to a license that asserts copyright protection is at best creating a confusing legal situation.
2. Another widely held misconception is to confuse a legal instrument designed for enforcement in a court of law with community norms enforced by accepted and community-policed community ethics. The latter is not somehow a weak mechanism. For example, there is no law that prohibits plagiarism in science, it is scientific ethics that does. Many scientists lean to use CC-BY and similar CC licenses not because they believe they have copyright in their data and are ready defend those in court, but to convey the wish to be attributed. Licenses are a poor instrument to achieve this - CC-BY for example makes no stipulation whatsoever as to _how_ attribution has to look like, even though we all know that to meet a scientists wish to be attributed, only certain formal ways will do.
3. The problem of license stacking, and as comes with it, the stacking of other legal obligations, such as attribution is a very real one in a world in which data analysis in increasingly done integratively, using an aggregate of data the components of which are themselves likely aggregates, and so on. We deliberately choose to ignore the legal obligation to properly attribute data sources in exhaustive recursion, but the obligation, if we are to take the licenses serious, is _real_ and _could_ subject people to lawsuits if not followed. If we are not to take licenses serious, then why apply them in the first place?

Bottom line, any data licensing article that talks about licensing also needs to talk about waiving copyright and how to do so (such as by applying a CC-Zero public domain waiver).

**Comment by elliewix** [comment]
Hi Hilmar,

Thanks for the comments.

There are a ton of nitty gritty details when it comes to data licensing, and I was hoping to keep it short.  So naturally things were omitted or not dwelled on.  The blog post is meant to provide a starting point for researchers to jump off of into their own research and not be the sole datasource.

I believe I can work in a statement about what of data can be copyrighted, and that's a good point.

I did hit on item #2 by discussing that CC0 does not excuse anyone from attribution, and you are still allowed to have a suggested citation or attribution method, both in content and form.

I did talk about CC0, but I made a deliberate point to not stress one or another because that is the choice of the dataset's authors.  It is not my place to dictate how restrictive or permissive an author license selection should be. They need to decide that based on their own community and desires.

**Comment by hlapp** [comment]
> I did talk about CC0, but I made a deliberate point to not stress one or another because that is the choice of the dataset's authors. It is not my place to dictate how restrictive or permissive an author license selection should be. They need to decide that based on their own community and desires.

Part of my point is that that's not enough. People need to look beyond their (sometimes small) communities, or what they perceive as such, because science is being (and should continue to be) de-siloed. Also, people may have less choice than your suggestion makes it out to be. If, for example, a journal requires data deposition in Dryad (and many do) then they are required to release their data under CC0. Many other (though not all) data repositories have this requirement, too.

**Comment by elliewix** [comment]
The repository issue is addressed in the "step one..." section, where I mention that individual repositories may have stock licenses to choose from.

**Comment by elliewix** [comment]
I've put in a PR proposing some clarifications about this commentary, but I'm not interested in expanding the blog post any further.  Discussion of the implications of CC0 and CCBY deserves its own blog post.
